writeengine/bulk/we_tableinfo.cpp

```diff
@@ -266,24 +271,44 @@ int TableInfo::readTableData()
 {
   RID validTotalRows = 0;
   RID totalRowsPerInputFile = 0;
+  int64_t totalRowsParquet = 0;   // totalRowsParquet to be used in later function
+                                  // needs int64_t type
   int filesTBProcessed = fLoadFileList.size();
   int fileCounter = 0;
   unsigned long long qtSentAt = 0;
-
-  if (fHandle == NULL)
+  if (fImportDataMode != IMPORT_DATA_PARQUET)
```

Comment:
Ideally this condition(and all similar ones) should become separate classes with a common parent.